dtu-hpc-python3/setup-python3.sh

```diff
@@ -246,6 +246,10 @@ export PYTHON_BIN_PATH=`which python3`
 export TF_NEED_GCP=0
 export TF_NEED_HDFS=0
 export TF_NEED_CUDA=1
+export TF_NEED_JEMALLOC=0
+export TF_NEED_GCP=0
```

Comment:
I just assumed I wouldn't be needing OpenCL because DTU only uses Nvidia GPU's 😛